packages/trpc/server/routers/viewer/organizations/adminGetUnverified.handler.ts

```diff
@@ -0,0 +1,51 @@
+import { prisma } from "@calcom/prisma";
+
+import type { TrpcSessionUser } from "../../../trpc";
+
+type AdminGetUnverifiedOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+};
+
+export const adminGetUnverifiedHandler = async ({ ctx }: AdminGetUnverifiedOptions) => {
```

Comment:
Get all unverified orgs - see loom for demo.

This is used in ADMIN only calls